react/features/overlay/actions.js

```diff
@@ -62,3 +65,36 @@ export function suspendDetected() {
         type: SUSPEND_DETECTED
     };
 }
+
+/**
+ * Discards any fatal error currently stored.
+ *
+ * @returns {{
+ *     type,
+ *     fatalError: undefined
+ * }}
+ */
+export function discardFatalError() {
```

Comment:
why not have a single `setFatalError` action, which is called with an error or undefined?

Reply:
I thought it's more readable this way, but I can change it to one action